rpcs3/rpcs3_app.cpp

```diff
@@ -88,6 +91,11 @@ void rpcs3_app::Init()
 		welcome_dialog* welcome = new welcome_dialog();
 		welcome->exec();
 	}
+#ifdef WITH_DISCORD_RPC
+	DiscordEventHandlers handlers;
+	memset(&handlers, 0, sizeof(handlers));
```

Comment:
Is there any particular reason to use `memset` here rather than simply default-initialising with `DiscordEventHandlers handlers = {}`, much more C++ and concise form?

Reply:
copy pasta from official discord stuff. Will fix.